Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM elixir:1.4.2
+
+ENV NODE_VERSION=7
+
+# Install system dependencies
+RUN apt-get -y update \
+    && apt-get -y install apt-transport-https build-essential curl git make locales locales-all inotify-tools
+
+# Install nodejs
+RUN curl -sL https://deb.nodesource.com/setup_$NODE_VERSION.x | bash - \
+    && apt-get -y install nodejs
+
+# Clean up apt temporary apt artefacts
+RUN apt-get clean -y \
+    && rm -rf /var/cache/apt/*
```

Comment:
i think if they don't change too often it's nice if you chain them together. With separating into new lines how you did already 👍